TestGraphGenerator/DbTranformations/DatabaseGenerator.py

```diff
@@ -0,0 +1,25 @@
+import random
+from typing import List, AnyStr
+
+from Config import GlobalSettings
+from TestGraphGenerator.Models import Node, Relationship
+from TestGraphGenerator.Models.Database import Database
+from TestGraphGenerator.Models.Graph import Graph
+
+
+def create_graph_map(names: List[AnyStr]) -> Database:
+    """
+    Creates a randomly generated graph in neo4j and saves it to a json representation for use in testing the differ
+    loads names for generating from a file.
+    :param names: the path to save the graph to.
+    """
+    names = names
+    relationships = []
+    names_nodes = [Node(GlobalSettings.TEST_GRAPH_TYPE, properties={"name": name}) for name in names]
+    for current_node in names_nodes:
+        for i in range(random.randint(1, 5)):
+            selected_node = random.choice(names_nodes)
+            if selected_node is not current_node:
```

Comment:
Are relationships bi-directional?
If so, you need to check that no connection exists as well

Reply:
done in  the graph engine